.github/workflows/codeql-analysis.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           languages: java
 
       - name: Setup Java 11
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: 11
```

Comment:
IIRC this was a breaking change from v1 to v2, requiring distribution

```suggestion
        uses: actions/setup-java@v3
        with:
          distribution: temurin
          java-version: 11
```

Reply:
hm, can't apply suggestion b/c of branch protection, will close this and send PR from my fork